<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GraphView" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>700</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_graph"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_position"/>
    <addaction name="actionSave_position"/>
    <addaction name="separator"/>
    <addaction name="actionImport_kmers"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionFind_Euler_Path"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_graph">
   <property name="text">
    <string>Load graph...</string>
   </property>
  </action>
  <action name="actionSave_graph">
   <property name="text">
    <string>Save graph...</string>
   </property>
  </action>
  <action name="actionLoad_position">
   <property name="text">
    <string>Load position...</string>
   </property>
  </action>
  <action name="actionSave_position">
   <property name="text">
    <string>Save position...</string>
   </property>
  </action>
  <action name="actionImport_kmers">
   <property name="text">
    <string>import kmers...</string>
   </property>
  </action>
  <action name="actionFind_Euler_Path">
   <property name="text">
    <string>Find Euler Path</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphView</class>
   <extends>QWidget</extends>
   <header>graphview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
